setup.cfg

```diff
@@ -70,7 +70,8 @@ addopts = --doctest-rst
 
 [flake8]
 max-line-length = 100
-select = E101,W191,W291,W292,W293,W391,E111,E112,E113,E502,E722,E901,E902
+# E402: Module level import not at top of file
+ignore = E402,W504,E226,F403,E126,E123,W503
```

Comment:
Having it alphanumerically ordered and have a list of comments to translate it makes it easier for future maintenance.

```suggestion
# E123: closing bracket does not match indentation of opening bracket's line
# E126: continuation line over-indented for hanging indent
# E226: missing whitespace around arithmetic operator
# F403: star import used; unable to detect undefined names
# W503: line break before binary operator 
# W504: line break after binary operator
ignore = E123,E126,E226,E402,F403,W503,W504
```

I would still advise against ignoring F403. It is safer to not use it unless you are managing namespace; and even then, use `# noqa` at the end of each affected line in the code.

Reply:
Thanks, committed this. We were planning to follow up separately with a review of our __init__ files for F403 and F401, but I may have time to get to it in this PR. If not I'll create an issue to do so.